src/web/app/components/response-time-series-chart/response-time-series-chart.component.ts

```diff
@@ -40,10 +42,29 @@ export class ResponseTimeSeriesChartComponent implements OnInit {
     this.feedbackResponseStatsService.loadResponseStats(durationSeconds.toString(),
         this.model.intervalSeconds.toString())
         .subscribe((records: FeedbackResponseRecords) => {
-          console.log(records);
-          this.drawChart(records.responseRecords, durationSeconds * 1000);
+          const canvas: any = d3.select('svg');
+
+          // clear all content
+          canvas.selectAll('*').remove();
+
+          this.drawChart(canvas, records.responseRecords, durationSeconds * 1000, false, '#696969');
+
+          const temp: number[] = [];
+          records.responseRecords.forEach((record: FeedbackResponseRecord, i: number) => {
+            temp[i] = record.count;
+
+            if (i <= 0) {
+              record.count = 0;
+            } else {
+              record.count = record.count - temp[i - 1];
+            }
+
+            return record;
```

Comment:
forEach takes in a Consumer... I believe in 2030 we have learnt what a Consumer does. What does the return here mean?